exercises/luhn/example.el

```diff
@@ -4,7 +4,8 @@
 
 ;;; Code:
 
-(require 'seq)
+(require 'cl)
```

Comment:
`cl` is deprecated since 2013 (Emacs 24.3), people should use `cl-lib` and don't omit the `cl-` prefix.

Reply:
Good catch @xuchunyang. I think this is generally a sign that our CI setup good use some work overall.